packages/wdio-runner/src/index.ts

```diff
@@ -98,6 +98,16 @@ export default class Runner extends EventEmitter {
         this._specs = specs
         this._caps = caps
 
+        /*
+        * Checks tsconfig.json path, throws error if it doesn't exist
+        */
+
+        if (args.autoCompileOpts?.tsNodeOpts?.project) {
+            if (!fs.existsSync(args.autoCompileOpts.tsNodeOpts.project)) {
+                throw new Error('Path to tsconfig.json is incorrect in config file')
```

Comment:
`throw new Error('Path to tsconfig is incorrect in config. File ${ args.autoCompileOpts.tsNodeOpts.project) not found`

As tsconfig.json can have another names should the error be more specific